src/Filter/modules/DurationFilter/MinMaxInput.tsx

```diff
@@ -0,0 +1,128 @@
+import { FC, useEffect, useState } from 'react';
+
+import Input from 'src/Input/Input';
+import useDebounce from '../../../lib/hooks/useDebounce';
+
+export interface MinMax {
+    min?: number;
+    max?: number;
+}
+
+const validateNumber = (num: number, allowNegatives: boolean) => {
```

Comment:
Please move this to a util file called `durationFilterUtil.ts` and add a unit tests for this function.